examples/test-client/package.json

```diff
@@ -1,47 +1,39 @@
 {
-  "name": "test-client",
+  "name": "test-app",
```

Comment:
```suggestion
  "name": "test-zapp",
```

Reply:
also, could be good to have this deploy to `zapp.onrender.com` or something like that